### PR TITLE
Frontend

### PR DESCRIPTION
**Pull request by AnneEjsing**

### CODE REVIEW

File: WebServer/frontend/src/app/videoplayer/videoplayer.component.ts [hunk 1]
```diff
@@ -1,15 +1,89 @@
...
+    this.api.getDefaultMedia().subscriptions.play.subscribe(
+      () => {
+        if(this.api.isLive){
+          this.api.seekTime(100, true);
+        }
+      }
+    )
+    // //If not a live video just start playing asap
+    // //If live this will make it a few seconds behind idk why so you have to manually press play.
+    // this.api.getDefaultMedia().subscriptions.canPlay.subscribe(
```
Comment (AnneEjsing) [inline WebServer/frontend/src/app/videoplayer/videoplayer.component.ts:38]: delete

File: WebServer/frontend/src/app/videoplayer/videoplayer.component.ts [hunk 2]
```diff
@@ -1,15 +1,89 @@
...
+        );
+    })
+  }
+
+  setBitrate(option: BitrateOption) {
+    if (this.currentStream.type == "hls") this.vgHls.setBitrate(option);
   }
 
-  ngOnInit() {
+  // onClickStream(stream: IMediaStream) {
```
Comment (AnneEjsing) [inline WebServer/frontend/src/app/videoplayer/videoplayer.component.ts:69]: delete

File: WebServer/frontend/src/app/videoplayer/videoplayer.component.ts [hunk 3]
```diff
@@ -1,15 +1,89 @@
...
+    )
+    // //If not a live video just start playing asap
+    // //If live this will make it a few seconds behind idk why so you have to manually press play.
+    // this.api.getDefaultMedia().subscriptions.canPlay.subscribe(
+    //   () => {
+    //     if(!this.api.isLive){
+    //       this.api.play();
+    //     }
+    //   }
+    // )
```
Comment (lrecht) [inline WebServer/frontend/src/app/videoplayer/videoplayer.component.ts:44]: delete?

File: WebServer/frontend/src/app/videoplayer/videoplayer.component.ts [hunk 4]
```diff
@@ -1,15 +1,89 @@
...
+  //     this.api.pause();
+  //     this.bitrates = null;
+
+  //     let t: Subscription = timer(0, 10).subscribe(
+  //         () => {
+  //             this.currentStream = stream;
+  //             t.unsubscribe();
+  //         }
+  //     );
+  // }
```
Comment (lrecht) [inline WebServer/frontend/src/app/videoplayer/videoplayer.component.ts:79]: delete?

File: WebServer/microservices/receiver/video_file_saver.py [hunk 5]
```diff
@@ -1,66 +1,71 @@
...
+# name = "test"
+# i = 0
+#
+# sp.call("mkdir test", shell=True)
+# for i in range(2):
+#    sp.call("ffmpeg -i "+ stream +" -r 25 -t 30 "+'test/'+name+str(i)+".mp4;", shell=True)
+#    sp.call("echo file \'"+name+str(i)+".mp4\' >>  test/input.txt", shell=True)
+#
+# sp.call("ffmpeg -y -f concat -safe 0 -i 'test/input.txt' -c copy test/output.mp4" ,shell=True)
+# sp.call("rm test/test* test/input.txt", shell=True)
```
Comment (lrecht) [inline WebServer/microservices/receiver/video_file_saver.py:19]: delete?

**Pull request merged**